contracts/governance/GoodDollarStaking.sol

```diff
@@ -234,27 +234,52 @@ contract GoodDollarStaking is
 		_setAPY(_interestRatePerBlock);
 	}
 
-	/// @dev helper function for multibase
-	function getRewardsPerBlock() public view returns (uint256) {
-		return rewardsPerBlock[address(this)];
+	/// @dev helper function for multibase and FixedAPYRewards
+	function getRewardsPerBlock() public view returns (uint256 _goodRewardPerBlock, int128 _gdRewardPerBlockX64) {
```

Comment:
```suggestion
	function getRewardsPerBlock() public view returns (uint256 goodRewardPerBlock, uint256 gdInterestRatePerBlock) {
```
prefix `_` is only for method parameters
you should return the interestrate in regular uint256, that means that you need to multiply interestrateX64 by precision (`.mulu`)